packages/riipen-ui/src/components/Chip.test.jsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import Adapter from "enzyme-adapter-react-16";
+import { shallow, mount, configure } from "enzyme";
+import toJson from "enzyme-to-json";
+
+import Chip from "./Chip";
+
+configure({ adapter: new Adapter() });
+
+describe("Component: Chip", () => {
+  it("shallow renders succeessfully", () => {
+    shallow(<Chip />);
+  });
+
+  it("gets default props and renders correctly", () => {
+    const wrapper = mount(<Chip />);
+    const defaultProps = new Chip().type.defaultProps;
+    defaultProps.classes.push("riipen-chip", "riipen");
+    expect(
+      wrapper
+        .find("Chip")
+        .props()
+        .classes.sort()
+    ).toEqual(defaultProps.classes.sort());
+
+    const component = wrapper.find("Chip").childAt(0);
+    expect(component.name()).toEqual("div");
+    expect(component.hasClass(defaultProps.color)).toEqual(true);
+    expect(component.hasClass("disabled")).toEqual(false);
+    expect(component.hasClass("hover")).toEqual(false);
+    expect(component.hasClass(defaultProps.size)).toEqual(true);
+    expect(component.hasClass(defaultProps.variant)).toEqual(true);
+    expect(component.childAt(0).name()).toEqual("span");
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+    wrapper.unmount();
+  });
+
+  it("displays a given label through the label prop", () => {
+    const label = "A test label";
+    const wrapper = mount(<Chip label={label} />);
+    expect(wrapper.find("Chip").props().label).toEqual(label);
+    expect(
+      wrapper
+        .find("Chip")
+        .find("span")
+        .text()
+    ).toEqual(label);
+    wrapper.unmount();
+  });
+
+  it("displays a label given as a child of Chip", () => {
```

Comment:
Again for naming convention, you could do:


```js
describe('label prop', () => {
  it('should display label from child', () => {
    ...
  });
});
```

Try to apply this philosophy to all remaining tests in this PR.